lms/static/scripts/frontend_apps/services/api.js

```diff
@@ -0,0 +1,28 @@
+import { apiCall } from '../utils/api';
+
+/**
+ * Low-level LMS backend API client service.
+ */
+export class APIService {
```

Comment:
There are three very small services here that handle API calls to the backend. One low-level one and two higher-level ones. This means that each one is quite focused, but in more code for the same functionality than we had before. Given that the methods are all very simple, another option would be to combine them all into one service.